### DISCUSSION TITLE
Blogging monthly challenge

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/virtualcoffee.io#458] [Monthly Challenge]
In November, we'll be running our second annual blogging challenge where we try to hit 50,000 words as a group. 

We'd love to have a list of topic suggestions to help folks who are working through writers block. 

If you have some topic suggestions, please drop them below 👇 and check back for the challenge page on Nov. 1st!

**Comment by danieljanderson** [comment]
I learning Java so I might be able to do a couple of things.  Comparing Java to Node/JavaScript.  I might be able to write about possible issues that might come up doing a Java and React stack.

**Comment by jdwilkin4** [comment]
Here are some topics that people can write about. 

How to deal with imposter syndrome as a developer?
How deal with burnout when applying for your first developer job?
What did you learn through your first developer interview?
What did you wish you knew before starting your first developer job?
What was your experience like during the first few months of your junior developer job?
How did you overcome the fear of building in public? What tips do you have for developers who want to start building in public? 
How to push through with learning how to code when learning was difficult at times?
How to start a freelancing career?

**Comment by meg-gutshall** [comment]
I like quick bite blog posts, like a 3–5 minute read, because a lot of times if it's any longer than that, I end up bookmarking it to read later and never go back to it.

Tiny blog post ideas:
* Write about a function or a method --> What it does, when you should use it, and why
* A time you successfully troubleshooted (troubleshot?) an error --> This can be **super** helpful for those who copy/paste an error message
* A simple script in your terminal
* A customization that saves you time/frustration in your daily workflow

Comment with your ideas!

**Comment by ClJarvis** [comment]
Keep a journal of your coding journey and then pull topics from it. Keep a brag board of your successes, or a Time I learned log.  Then write about one of your favorites.

**Comment by sadiejay** [comment]
A meta post like how did you chose where you blog, other options your were considering before where you're currently set up. Why are you blogging? How was the set up process? Features/topics/platforms you're thinking to incorporate in the future.

**Comment by ajcwebdev** [comment]
For me I like picking some combination of framework and hosting/deployment provider and figuring out and documenting how to generate a boilerplate project, slightly modify it to put your own stamp on it, and then deploy it to a popular hosting provider. So take some metaframework from the popular list of options:
* Next
* Gatsby
* Redwood
* Blitz
* Nuxt
* Gridsome
* Elder
* SvelteKit
* Plenti

And combine them with any popular hosting provider:
* Netlify
* Vercel
* Layer0
* Fly
* Qovery

And bonus points for adding a database:
* Railway
* Supabase
* Nhost
* Hasura
* PlanetScale
* CockroachDB

I like this because it's a great way to learn a couple pieces of tech at once, and also can provide useful documentation both for an open source project and a company that wants to support open source projects. It also gives you essentially hundreds of different combinations of potential topics to write about.
